lib/features/editor-actions/BpmnEditorActions.js

```diff
@@ -51,6 +51,9 @@ BpmnEditorActions.prototype._registerDefaultActions = function(injector) {
   var directEditing = injector.get('directEditing', false);
   var searchPad = injector.get('searchPad', false);
   var modeling = injector.get('modeling', false);
+  var popupMenu = injector.get('popupMenu', false);
+  var contextPad = injector.get('contextPad', false);
+  var contextPadProvider = injector.get('contextPadProvider', false);
```

Comment:
This feels a little "hacky" for me, for two reasons:

* We depend on `contextPadProvider`
* We depend on internal APIs exposed by context pad provider

I think the first one is fine, but let's expose `open popup menu` as a dedicated API on the context pad provider.